ecosystem/platform/server/app/controllers/application_controller.rb

```diff
@@ -27,7 +27,9 @@ def after_sign_in_path_for(user)
     stored_location = stored_location_for(user)
     return stored_location if stored_location.present?
 
-    if user.email.nil?
+    if !user.authorizations.where(provider: :discord).exists?
+      root_path
```

Comment:
this is supposed to be the IT1 registration path, no?

Reply:
This is so that if the sign in with GitHub on [this screen](https://user-images.githubusercontent.com/310773/169143176-14f99e28-6f66-4dd1-b62b-a7152f1cd919.png), they then will see [this screen](https://user-images.githubusercontent.com/310773/169143175-fbde620b-a8e9-4126-bca7-b111688c91ff.png).